apps/morph_stats

```diff
@@ -84,32 +81,38 @@ def get_parser():
                         help=('Summary output file name, if it ends in .json, '
                               'a json file is created, if .csv, then a csv file'))
 
+    parser.add_argument('-I', '--ignored-exceptions', dest='ignored_exceptions', default='',
```

Comment:
I'll make the doc better.  I don't think choices handles python classes, does it?